src/utils.js

```diff
@@ -0,0 +1,51 @@
+const { DEFAULT_PAGE, DEFAULT_PER_PAGE } = require('./vars');
+
+const buildQuery = (base, query) => {
+  const builtQuery = buildQueryString(base, query);
+  const sqlParams = buildParams(query);
+  console.log(builtQuery, sqlParams);
+  return [builtQuery, sqlParams];
+};
+
+const buildParams = query => {
+  const sqlParams = [];
+
+  for (const param in query) {
+    if (param === 'page' || param === 'per_page') {
+    } else sqlParams.push(query[param]);
```

Comment:
This can be confusing to read, I'd recommend a tweak to get rid of the `else` here

```js
if (param !== 'page' && param !== 'per_page') {
  sqlParams.push(query[param]);
}
```